website/next.config.mjs

```diff
@@ -32,14 +32,15 @@ export default withGuildDocs({
       '/docs/library/author-load-config': '/docs/library/load-config',
       '/docs/library/author-loaders': '/docs/library/loaders',
       '/usage': '/docs/user/usage',
-      '/docs/user/user-usage': '/docs',
-      '/docs/user': '/docs',
+      '/docs/user/user-usage': '/docs/user/usage',
+      '/docs/user': '/docs/user/usage',
       '/docs/introduction': '/docs',
-      '/docs/user/user-schema': '/docs',
-      '/docs/user': '/docs',
       '/extensions': '/docs/library/extensions',
       '/introduction': '/docs',
       '/schema': '/docs/user/schema',
+      '/docs': '/docs',
+      '/docs/schema': '/docs/user/schema',
+      '/docs/user/null': '/docs/user/usage',
```

Comment:
I think we shouldn't handle the `/null`, usually it's just weird scanner tools that causes it. Please remove.